packages/frontend/src/scripts/configureTooltips.ts

```diff
@@ -95,6 +95,12 @@ export function configureTooltips() {
 
     element.addEventListener('click', (e) => {
       e.stopPropagation()
+      const isDisabledOnMobile = element.getAttribute(
+        'data-tooltip-mobile-disabled',
+      )
```

Comment:
Extract it outside event listener. Best would be 85 line I guess.

Also the best would be to pass isDisabledOnMobile to `show` function.